app/scripts/controllers/swaps.js

```diff
@@ -309,18 +313,15 @@ export default class SwapsController {
     })
   }
 
-  async setInitialGasEstimate (initialAggId, baseGasEstimate) {
```

Comment:
This value is still being passed in in the sole place this function is invoked

Reply:
Specifically the `quote.maxGas` value is being passed in.

What is `quote.maxGas` :thinking: ? Would it be OK for the estimate to exceed the `quote.maxGas` value?

Reply:
addressed in 57ad56a5d

Reply:
The answer to my question was that `quote.maxGas` is the API's estimate of the max gas limit to expect, and it's perfectly normal for it to differ from the estimate the client gets. The API may use a different method to estimate gas, and network conditions may have changed since that estimate was calculated.